src/infrastructure/remote/login-user.ts

```diff
@@ -14,5 +17,57 @@ export function loginUserRemote(): LoginUserService {
       accessToken: token,
     };
   };
-  return { getUserInfo };
+
+  const postLogin = async (authorizationCode: string) => {
+    const response = await publicAPI
+      .post({
+        url: `/auth/login`,
+        data: { authenticationCode: authorizationCode, provider: 'kakao' },
+      })
+      .catch((error: AxiosError) => {
+        if (error.response?.status === STATUS_CODE.BAD_REQUEST)
+          throw new NotFoundError('로그인에 실패하였습니다.');
+      });
+    const { id, profileId, name, image, refreshToken } = response.data.user;
+    return {
+      isJoined: profileId.length > 0 && name.length > 0,
```

Comment:
```suggestion
      isJoined: profileId.length && name.length,
```
`> 0` 없어도 똑같이 동작할 것 같아요 !!

Reply:
이미 머지 해버렸지만...... 이렇게 하면 isJoined에 number가 들어가서 안됩니다 !! 저도 이걸 잊고 그대로 반영해버렸는데 https://github.com/Neogasogaeseo/Naega-Web/issues/408 에서 고칠게용